src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -254,7 +254,7 @@
     <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>{{ translation_domain is same as(false) ? label|raw : label|raw }}
       {% if label_attr['tooltip'] is defined %}
         {% set placement = label_attr['tooltip_placement'] is defined ? label_attr['tooltip_placement'] : 'top' %}
-        <i class="icon-question" data-toggle="pstooltip" data-placement="{{ placement }}"
+        <i class="help-box" data-toggle="pstooltip" data-placement="{{ placement }}"
```

Comment:
should it be `icon-question-sign` instead? same in `admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl`

Reply:
No, `admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl` is used with Bootstrap 3, but the twig layout is used with new-theme and uses bootstrap 4 which doesn't include any icon library anymore : https://getbootstrap.com/docs/4.0/extend/icons/

Reply:
You're right. But in the new-theme we also use material icons.
Maybe time to use `<i class="material-icons">help</i>` instead?

Reply:
We could, but why we don't use the same class as everywhere

![image](https://user-images.githubusercontent.com/5637724/68843892-278d8280-06c9-11ea-9717-c223d5b3c587.png)